### DISCUSSION TITLE
Air Quality measurement

### DISCUSSION THREAD

**Discussion by JonFear** [Daveiano/weewx-wdc#143] [General]
Hi Dave

I am sure I have missed it somewhere but where do I enable the output of my WH41 Air Quality sensor?

BW

Jon

**Comment by Daveiano** [comment]
The WH41 is measuring `pm2_5`, right?

If so, you can just add `pm2_5` to `stat_tile_observations` in the skin.conf.

If you want to add this also as a chart you have to do some additional steps, and I just noticed that there is no wiki page to address this. 

1. Define the defaults here: https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/skin.conf#L268
2. Add `pm2_5` as desired for day, week, month, ... here https://github.com/Daveiano/weewx-wdc/blob/3.x/skins/weewx-wdc/skin.conf#L365

Also: I think I should add defaults for the `pm` observations.

**Reply by JonFear** [reply]
Thanks Dave, something not quite right though... The tile shows and has the days max but the current shows N/A...

Jon

**Reply by Daveiano** [reply]
Ok, that's hard to resolve for me without any clues. Did you enable it only for the stat tiles or also for as chart? 

Some hints: 

- Does it work on the week/month/year pages?
- I assume you are storing the data from the WH41 directly to the default weewx db (default data binding)?
- Are you using live updates via MQTT?

**Comment by JonFear** [comment]
OK Dave, this appears to be working correctly now. I am closing this one off...
